core/src/saros/misc/xstream/XStreamExtensionProvider.java

```diff
@@ -99,6 +100,27 @@ public XStreamExtensionProvider(String namespace, String elementName, Class<?>..
 
     xstream.registerConverter(BooleanConverter.BINARY);
     xstream.registerConverter(new UrlEncodingStringConverter());
+
+    /*
+     * set map conversion format to
```

Comment:
not necessary, but because this pr breaks compatibility anyway a chance to increase readability of our xmpp messages and make them shorter

so already changing this breaks the whole compatibility because old `VersionExchangeExtension` couldn't be parsed